cli/testdata/coder_speedtest_--help.golden

```diff
@@ -4,11 +4,11 @@ Usage:
   coder speedtest <workspace> [flags]
 
 Flags:
-  -d, --direct          Specifies whether to wait for a direct connection before testing speed.
-  -h, --help            help for speedtest
-  -r, --reverse         Specifies whether to run in reverse mode where the client receives and
-                        the server sends.
-  -t, --time duration   Specifies the duration to monitor traffic. (default 5s)
+  -d, --direct             Specifies whether to wait for a direct connection before testing speed.
+      --direction string   Specifies whether to run in reverse mode where the client receives
```

Comment:
```suggestion
      --direction string   Specifies whether to test downlink or uplink throughput (up|down) (default "down")
```

Reply:
The CLI flag renderer should really be doing this.